lxc/action.go

```diff
@@ -3,40 +3,63 @@ package main
 import (
 	"fmt"
 
+	"github.com/codegangsta/cli"
```

Comment:
We keep core, external and internal imports separated in groups throughout our codebase, as codegangsta isn't part of lxd's codebase, please add a newline after it.